corehq/apps/campaign/views.py

```diff
@@ -197,11 +197,13 @@ def save_widget(self, request, *args, **kwargs):
 
         widget = model_class(dashboard=self.dashboard)
         form = form_class(self.domain, request.POST, instance=widget)
+        show_success = False
         if form.is_valid():
             form.save(commit=True)
+            show_success = True
```

Comment:
If you do ...
```suggestion
            form = form_class(self.domain)
```
... then you don't need that `if` statement in `context`.

Reply:
https://github.com/dimagi/commcare-hq/pull/35995/commits/de265af0cc977898a1174b4b35c06bec41e33b07